Packs/AbnormalSecurity/ReleaseNotes/2_1_2.md

```diff
@@ -1,3 +1,3 @@
 #### Integrations
 ##### Abnormal Security
-- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
```

Comment:
not sure whats going on here? seems there is no change

Reply:
You need to add new Release Notes, you edited the `Packs/AbnormalSecurity/ReleaseNotes/2_1_2.md`
file that already exist.

Reply:
Yeah I just can't revert it, if you see the diff there is actually no change 

- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
- Updated the Docker image to: *demisto/python3:3.10.13.80014*.

its the exact same line 🤔

Reply:
can't figure out how to do this cleanly - can we keep this change given its not actually changing anything @sapirshuker ?